pydis_site/apps/content/resources/guides/pydis-guides/contributing/pull-requests.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Pull Requests
+description: A guide for opening pull requests.
+---
+
+As stated in our [Contributing Guidelines](../contributing-guidelines/), do not open a pull request if you aren't assigned to an approved issue. You can check out our [Issues Guide](../issues/) for help with opening an issue or getting assigned to an existing one.
+{: .notification .is-warning }
+
+
```

Comment:
I assume the double-newlines are for this notification thing?

Reply:
Reduced to 1